lib/plugins/aws/package/compile/functions.js

```diff
@@ -414,6 +415,22 @@ class AwsCompileFunctions {
       functionResource.Properties.Layers = this.serverless.service.provider.layers;
     }
 
+    if (this.console.isEnabled && this.console.isFunctionSupported(functionObject)) {
+      if (!functionResource.Properties.Environment) functionResource.Properties.Environment = {};
+      if (!functionResource.Properties.Environment.Variables) {
+        functionResource.Properties.Environment.Variables = {};
+      }
+      if (!functionResource.Properties.Layers) functionResource.Properties.Layers = [];
+
+      Object.assign(
+        functionResource.Properties.Environment.Variables,
+        await this.console.deferredFunctionEnvironmentVariables
+      );
+      functionResource.Properties.Layers.push({
```

Comment:
Should we validate if that layer can actually be added ? I believe there's a limit of 5 layers per function, what if someone is already using 5 of them?

Reply:
Good point, I haven't thought about that. Due to limited timeframe, I won't address that now, but add it into TODO list